queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/alternate_name.py

```diff
@@ -62,48 +65,49 @@ def update_partner_change(
             end_date=None,
             identifier=legal_entity.identifier,
             name=to_legal_name,
-            name_type=AlternateName.NameType.OPERATING,
-            start_date=start_date,
-            registration_date=change_filing_rec.effective_date,
+            name_type=AlternateName.NameType.DBA,
+            start_date=alternate_name.start_date,
+            business_start_date=business_start_date,
+            entity_type=BusinessCommon.EntityTypes.PARTNERSHIP,
         )
         legal_entity.alternate_names.append(new_alternate_name)
 
-        filing_meta = {
-            **filing_meta,
-            "fromLegalName": alternate_name.name,
-            "toLegalName": to_legal_name,
-        }
+        filing_meta.update(
+            {
+                "fromLegalName": alternate_name.name,
+                "toLegalName": to_legal_name,
+            }
+        )
```

Comment:
so are you able to verify updating the existing meta data this way works?  i.e. the changes are picked up properly in db.

In the past I seem to recall that if you don't replace the meta data entirely, there would be issues.  But that issue may not exist anymore with all the updates we've made.

Reply:
Yes it can be verified. At least in change of registration filing processor, meta data didn't get updated by replacing the data. From what I can see, it just rebinds the variable to a new dict reference inside the method and the original one outside the method won't be affected.